demo/server/src/app.ts

```diff
@@ -17,17 +19,23 @@ app.use(logger(config.morganOptions));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(path.join(__dirname, '../public')));
 app.use(favicon(path.join(__dirname, '../public', 'favicon.ico')));
 
-app.use('/', index);
+app.use('/', index, security);
 
 app.use((req: express.Request, res: express.Response, next) => {
   const err: any = new Error('Not Found');
   err.status = 404;
   next(err);
 });
 
+function setupCustomLoginPageRender() {
+  app.engine('hbs', expressHbs());
+  app.set('view engine', 'hbs');
+}
+setupCustomLoginPageRender();
```

Comment:
This looks strange. Idea was to pull out login related code from entire file. 
This way it's even worse than before.